src/java.base/share/classes/jdk/internal/loader/URLClassPath.java

```diff
@@ -802,37 +799,20 @@ public Void run() throws IOException {
             }
         }
 
-        /* Throws if the given jar file is does not start with the correct LOC */
-        @SuppressWarnings("removal")
```

Comment:
I noticed the @SuppressWarnings("removal") when looking at the PR. It is added by https://github.com/openjdk/jdk/commit/6765f902505fbdd02f25b599f942437cd805cad1 for [JDK-8266459](https://bugs.openjdk.java.net/browse/JDK-8266459). So looks like the method has been targeted for deprcating/removal.

Reply:
Right, the suppression is for the use of `getSecurityManager` here.